packages/protocol/scripts/truffle/verify-bytecode.ts

```diff
@@ -26,20 +26,21 @@ const Registry: Truffle.Contract<RegistryInstance> = artifacts.require('Registry
 const Proxy: Truffle.Contract<ProxyInstance> = artifacts.require('Proxy')
 
 const argv = require('minimist')(process.argv.slice(2), {
-  string: ['build_artifacts', 'proposal', 'initialize_data', 'network'],
+  string: ['build_artifacts', 'proposal', 'initialize_data', 'network', 'libraries'],
   boolean: ['before_release_1'],
 })
 
 const artifactsDirectory = argv.build_artifacts ? argv.build_artifacts : './build/contracts'
 const network = argv.network ?? 'development'
 const proposal = argv.proposal ? readJsonSync(argv.proposal) : []
 const initializationData = argv.initialize_data ? readJsonSync(argv.initialize_data) : {}
+const libraries = argv.libraries ?? 'libraries.json'
```

Comment:
Nit: maybe rename to `librariesFilename`, otherwise looks like an object containing libraries?